core/shared/src/main/scala/cats/effect/unsafe/IORuntimeBuilder.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020-2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+package unsafe
+
+import scala.concurrent.ExecutionContext
+
+final class IORuntimeBuilder protected (
+    protected var computeWrapper: ExecutionContext => ExecutionContext = identity,
+    protected var blockingWrapper: ExecutionContext => ExecutionContext = identity,
+    protected var customConfig: Option[IORuntimeConfig] = None,
+    protected var customScheduler: Option[Scheduler] = None,
+    protected var customShutdown: Option[() => Unit] = None
+) extends IORuntimeBuilderPlatform {
+  def withComputeWrapper(wrapper: ExecutionContext => ExecutionContext) = {
```

Comment:
This is good for your usecase :) can we also add options for someone to completely replace the compute and blocking pools?

Reply:
Sure thing, done :wink:

Reply:
```suggestion
  def transformCompute(wrapper: ExecutionContext => ExecutionContext) = {
```

Maybe? Naming things is hard 😆

Reply:
Much better :wink: